src/conversationLogger.js

```diff
@@ -1,92 +1,99 @@
+// @flow
 // A knockoff of Dashbot's API for generic conversation logging
 const dashbot = require('dashbot');
 const winston = require('winston');
 const Slack = require('winston-slack-transport');
 
-const logger = new (winston.Logger)({transports: []});
+class ConversationLogger {
+  /*:: options: Object */
+  constructor({dashBotKey, logFile, slackChannel, slackWebhookUrl} = {}) {
+    this.logger = new (winston.Logger)({transports: []});
```

Comment:
while this line is changing, want to fix my mistake? I'm pretty sure those parens are extra
```
this.logger = new winston.Logger({transports: []});
```